src/Orleans/Runtime/RequestContextUtil.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace Orleans.Runtime
+{
+    public class RequestContextUtil
```

Comment:
I'm trying to wrap my head around the purpose of `RequestContextUtil`. What is `Export()` supposed to do eventually? Deep-copy `RequestContext`?